docs/UserGuide.md

```diff
@@ -1479,7 +1478,7 @@ Format: `createconsult d/DATE tt/TIME n/STUDENT_NAME n/STUDENT_NAME …`
 <br>
 
 #### 📝️Adding students to a consultation: `addtoconsult`
-You can add your student(s) into a consultation slot.
+You can add your student(s) into a consultation slot if you want to schedule or allocate specific individuals for the consultation.
```

Comment:
You can add your student(s) into a consultation slot if you `need` to schedule or allocate specific individuals for `your` consultation.